### PR TITLE
PLANET-7550: Move Deep Dive Topic pattern block to theme

### PR DESCRIPTION
**Pull request by mardelnet**
**Ref:** https://jira.greenpeace.org/browse/PLANET-7550

**Description:** This PR adds the "Deep Dive Topic" pattern block to the master theme.

**Related PR:** https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/pull/1228

### CODE REVIEW

**Comment by planet-4** [comment]
### Test instance is ready :rocket:

:new_moon: [nix](https://www-dev.greenpeace.org/test-nix) | [admin](https://www-dev.greenpeace.org/test-nix/wp-admin/) | [blocks report](https://www-dev.greenpeace.org/test-nix/wp-admin/?page=plugin_blocks_report) | [CircleCI](https://app.circleci.com/pipelines/github/greenpeace/planet4-test-nix) | [composer-local.json](https://github.com/greenpeace/planet4-test-nix/blob/main/composer-local.json)

:watch: 2024.08.02 09:14:49

File: assets/src/block-templates/deep-dive-topic/template.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    }],
+    ['planet4-block-templates/side-image-with-text-and-cta', {
+      title: __('What can be done', 'planet4-blocks'),
+      mediaPosition: 'right',
+    }],
+    ['planet4-blocks/covers', {
+      title: __('How you can help', 'planet4-blocks'),
+      cover_type: 'take-action',
+    }],
+    ['planet4-blocks/articles', {
```
Comment (mleray) [inline assets/src/block-templates/deep-dive-topic/template.js:21]: We should keep in mind that the Articles and Covers blocks haven't been moved to the theme, so right now if the plugin is disabled these do not work in the template... However I see that we have already moved several other patterns to the theme that use the Covers block, so it's probably not an issue for now.

File: assets/src/block-templates/deep-dive-topic/block.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+    "$schema": "https://schemas.wp.org/trunk/block.json",
+    "apiVersion": 2,
+    "name": "planet4-block-templates/deep-dive-topic",
+    "title": "Deep Dive Topic",
+    "category": "planet4-block-templates",
+    "icon": "editor-table",
+    "textdomain": "planet4-blocks-backend"
+  }
```
Comment (mleray) [inline assets/src/block-templates/deep-dive-topic/block.json:9]: I think this red symbol means that we are missing an empty line at the end of the file

File: assets/src/block-templates/deep-dive-topic/template.js (hunk 1 above)
Comment (mardelnet) [reply assets/src/block-templates/deep-dive-topic/template.js:21]: Good catch, thanks!

File: assets/src/block-templates/deep-dive-topic/block.json (hunk 2 above)
Comment (mardelnet) [reply assets/src/block-templates/deep-dive-topic/block.json]: Fixed!

**Review by mleray — APPROVED** [review]
Looks good to me 👍

**Pull request merged**